Source/FieldSolver/SpectralSolver/SpectralSolverRZ.H

```diff
@@ -117,24 +96,8 @@ class SpectralSolverRZ
          *                        storing the three components of the current density
          * \param[in]     rho     unique pointer to MultiFab storing the charge density
          */
-        void CurrentCorrection ( std::array<std::unique_ptr<amrex::MultiFab>,3>& current,
-                                 const std::unique_ptr<amrex::MultiFab>& rho ) {
-             algorithm->CurrentCorrection( field_data, current, rho );
-        }
-
-        /**
-         * \brief Public interface to call the virtual function \c VayDeposition,
-         * declared in the base class SpectralBaseAlgorithmRZ and defined in its
-         * derived classes, from objects of class SpectralSolverRZ through the private
-         * unique pointer \c algorithm.
-         *
-         * \param[in,out] current Array of unique pointers to \c MultiFab storing
-         *                        the three components of the current density
-         */
-        void VayDeposition (std::array<std::unique_ptr<amrex::MultiFab>,3>& current)
-        {
-            algorithm->VayDeposition(field_data, current);
-        }
```

Comment:
@RemiLehe Thank you again for refactoring #1216. I think this interface for the function `VayDeposition` was removed by accident here.

Reply:
Oops! Thanks a lot for pointing this out! I just updated the PR.